stack.yaml.lock

```diff
@@ -0,0 +1,72 @@
+# This file was autogenerated by Stack.
```

Comment:
I'm not sure what these files are. I thought they were generated and temporary. Did you mean to commit this?

Reply:
Yes, commiting this makes sure builds are deterministic.

Reply:
Turns out that it doesn’t work well with testing multiple GHC versions so I’ve killed it again :woman_shrugging: